tests/test_cookiejar.py

```diff
@@ -272,26 +272,32 @@ async def test_domain_filter_ip_cookie_receive(cookies_to_receive) -> None:
     assert len(jar) == 0
 
 
-async def test_preserving_ip_domain_cookies(loop) -> None:
-    jar = CookieJar(unsafe=True)
-    jar.update_cookies(SimpleCookie(
-        "shared-cookie=first; "
-        "ip-cookie=second; Domain=127.0.0.1;"
-    ))
-    cookies_sent = jar.filter_cookies(URL("http://127.0.0.1/")).output(
-        header='Cookie:')
-    assert cookies_sent == ('Cookie: ip-cookie=second\r\n'
-                            'Cookie: shared-cookie=first')
-
-
-async def test_preserving_quoted_cookies(loop) -> None:
-    jar = CookieJar(unsafe=True)
+@pytest.mark.parametrize(
+    'cookies,expected,quote_bool', [
```

Comment:
using a tuple here improves readability
```suggestion
    ('cookies', 'expected', 'quote_bool'), [
```

Reply:
Fixed. Thank you.